pages/api/stories/index.ts

```diff
@@ -16,16 +16,24 @@ export const config = {
 const handler = nc<NextApiRequest, NextApiResponse>();
 
 handler.get(async (req, res) => {
+  const page = typeof req.query.page === "string" ? parseInt(req.query.page) : 0;
+  const storiesPerPage = 4;
+
   const stories = await db.story.findMany({
     orderBy: {
       createdAt: "desc",
     },
     include: {
       topics: true,
     },
+    take: storiesPerPage,
+    skip: page * storiesPerPage,
   });
 
-  return res.json({ stories });
+  const storiesCount = await db.story.count();
```

Comment:
I'm not sure if it's possible, but I wonder if there's a way to add the count into the `findMany` call? Not a huge deal if not.

Reply:
Well, probably not since we're handling pagination in the same call. Ignore me :D